src/Relude/Lifted/Concurrent.hs

```diff
@@ -25,17 +25,40 @@ module Relude.Lifted.Concurrent
 
          -- * STM
        , STM
-       , TVar
        , atomically
+       , STM.orElse
+       , STM.check
```

Comment:
`orElse` and `check` functions also have too generic name. When I see such functions in the code, I don't expect them to work with `STM` immediately, they could do anything and Hoogle search confirms that there're multiple different implementations for those names. In `relude` we don't want to introduce aggressive name conflicts.

Reply:
Removed!